ui/org.eclipse.pde.ui/src/org/eclipse/pde/internal/ui/pderesources.properties

```diff
@@ -2353,6 +2353,8 @@ NoLineTerminationResolutionCreate_description=Adds a line break at the end of th
 NoLineTerminationResolutionCreate_label=Add a line break after the header
 NoLineTerminationResolutionRemove_description=Removes all whitespace characters from the last line of the manifest
 NoLineTerminationResolutionRemove_label=Remove excess whitespace from the end of the manifest
+ExtraneousLineResolutionRemove_label=Remove extraneous lines
```

Comment:
Would be more clear to add "empty" before "lines".  That's even more explanatory then "extraneous"